most_free_time.rb

```diff
@@ -13,8 +13,9 @@
 =end
 
 # my solution seems passed all given tests, accoridng to the return values. 
-# but doesn't show in the system. 
-# need code review and refactoring???????????????????????
+# but in coderbyte, it didn't show i passed all tests. 
```

Comment:
@bhennes2  The code is able to return the correct time value, but has problem of format the result, not sure how to improve Line55-Line56, so it returns the right format.